chapter_convolutional-neural-networks/padding-and-strides_vn.md

```diff
@@ -254,7 +255,12 @@ By default, the padding is $0$ and the stride is $1$.
 In practice, we rarely use inhomogeneous strides or padding, i.e., we usually have $p_h = p_w$ and $s_h = s_w$.
 -->
 
-*dịch đoạn phía trên*
+Để đơn giản, khi số đệm trên chiều cao và chiều rộng của đầu vào lần lượt là $p_h$ và $p_w$, chúng ta gọi là phần đệm $(p_h, p_w)$.
+Đặc biệt khi $p_h = p_w = p$, đây là phần đệm $p$.
+Khi sải bước trên chiều cao và chiều rộng lần lượt là $s_h$ và $s_w$, chúng ta gọi là sải bước $(s_h, s_w)$.
+Đặc biệt khi $s_h = s_w = s$, đây là sải bước $s$.
```

Comment:
Chỗ này em nghĩ sửa lại như này sát nghĩa `Để đơn giản` hơn.
```suggestion
Để đơn giản, khi số đệm trên chiều cao và chiều rộng của đầu vào lần lượt là $p_h$ và $p_w$, chúng ta kí hiệu phần đệm là $(p_h, p_w)$.
Trường hợp đặc biệt khi $p_h = p_w = p$, ta kí hiệu phần đệm là $p$.
Khi sải bước trên chiều cao và chiều rộng lần lượt là $s_h$ và $s_w$, chúng ta kí hiệu sải bước là $(s_h, s_w)$.
Trường hợp đặc biệt khi $s_h = s_w = s$, ta kí hiệu sải bước là $s$.
```

Reply:
Cảm ơn bạn review bài của mình. Mình đồng ý dịch như thế này sẽ hay hơn. Mình đang muốn chờ một bạn nữa review để xem kí kiệu như ở đây có đồng nhất với toàn bộ cuốn sách hay không. Vì nếu cũng có ký hiệu tương tự ở các phần khác của cuốn sách có thể gây nhầm lẫn cho người đọc.

Reply:
Mình cũng thấy cách dịch của @cuongvng dễ hiểu hơn.